MVVM Module/Tests/Feed UI/FeedUISnapshotTests.swift

```diff
@@ -65,7 +65,7 @@ private class FeedLoaderStub: FeedLoader {
 
 private extension FeedViewController {
 	func display(errorMessage: String) {
-		fatalError("Must be implemented - follow the MVC solution as a guide")
+		(tableView.tableHeaderView as? ErrorView)?.show(message: errorMessage)
```

Comment:
It'd be best to access the error view via its property, like it was done in the MVP solution.

This way, the test doesn't need to know the error view is the table header view. So the test isn't coupled with the view hierarchy structure and you can change the structure without breaking the tests:

```suggestion
		errorView?.show(message: errorMessage)
```